### PR TITLE
set timezone to lastCommunicationDate string before parsing as date

### PR DESCRIPTION
**Pull request by marianosimone**
Investigating #2156, I discovered that there's no problem with the date format, except for the fact that the timezone is NOT specified, so it depends on the user's timezone.

Adding EDT to the `lastCommunicationTime` should be enough to ensure consistent results

### CODE REVIEW

**Comment by marianosimone** [comment]
@moollaza @xuv I don't see any problems with parsing the `-`separators in the date

**Comment by xuv** [comment]
@marianosimone There is a problem in Firefox and Chrome, at least. 
Ex in Firefox:
```
new Date('2015-09-07 02:20:53 PM')
Invalid Date
new Date('2015/09/07 02:20:53 PM')
Date 2015-09-07T18:20:53.000Z
```

And should we add Momentjs Timezone lib? http://momentjs.com/timezone/docs/

**Comment by marianosimone** [comment]
@xuv wired... I'm running Chrome 45.0.2454.85 (64-bit) on MacOS and get:

    > new Date('2015-09-07 02:20:53 PM')
    < Mon Sep 07 2015 14:20:53 GMT-0400 (EDT)

Anyway, if there IS a problem with other browsers/OS, I'm adding the fix.

As per Moment Timezone, I'm not sure if we need it. @moollaza?

**Comment by marianosimone** [comment]
@moollaza @xuv, oh... shoot. Actually, appending EDT won't work, as when we go back to EST it will be off by an hour.

Let me try again with the people at CitiBike to see if I can get a better solution for this

**Comment by xuv** [comment]
@marianosimone My bad. You're right. `new Date('2015-09-07 02:20:53 PM')` works in Chrome. But not in Firefox 40.0.3 on Linux.

And for the EDT/EST thing, that's why I was suggesting Moment Timezone. Maybe that would help solve this. Not sure.

**Comment by moollaza** [comment]
@marianosimone if CitiBike can adjust their date stamp to be in UTC, or at least specify the timezone offset that would be great. I'd like to avoid adding additional overhead if possible, but worst case I guess we'll have to use MomentTimezone

**Comment by marianosimone** [comment]
@moollaza agreed. I'm in contact with them now, and expect to have an answer pretty soon. I'll keep you guys updated

**Comment by moollaza** [comment]
@marianosimone any updates on this?

**Comment by marianosimone** [comment]
Unfortunately, I haven't heard back from the people behind the feeds yet... I'm trying again now! Will keep you posted

**Comment by marianosimone** [comment]
This is not a good solution, and I'm following up with the people from CitiBIke for a better one. Will follow up in the referenced bug report
